sympy/printing/octave.py

```diff
@@ -432,6 +435,18 @@ def _print_LambertW(self, expr):
         return "lambertw(" + args + ")"
 
 
+    def _nested_binary_math_func(self, expr):
+        args = self._print(expr.args[0])
+        args += ', %s' % self._print(expr.func(*expr.args[1:]))
+
+        return '{name}({args})'.format(
```

Comment:
since there is no branching longer (we dropped nest) we can acutally simplify the whole function body to this:
```
return '{name}({arg1}, {arg2})'.format(
    name=self.known_functions[expr.__class__.__name__],
    arg1=self._print(expr.args[0]).
    arg2=self._print(expr.func(*expr.args[1:]))
)
```

Reply:
Thanks @bjodah !
It has been implemented.